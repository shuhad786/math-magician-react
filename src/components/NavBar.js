/* eslint-disable */
import { Link } from 'react-router-dom';

const Navbar = () => {
    return (
    <div className="navBar">
      <div className="Logo">
        Math Magicians
      </div>
      <ul className="links">
        <Link className="link" to="/">Home</Link>
        <Link className="link" to="/calculator">Calculator</Link>
        <Link className="link" to="/quotes">Quotes</Link>
      </ul>
    </div>
    )
  }

export default Navbar;