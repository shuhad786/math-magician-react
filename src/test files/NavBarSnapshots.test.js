import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import NavBar from '../components/NavBar';

it('Navbar renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <NavBar />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
