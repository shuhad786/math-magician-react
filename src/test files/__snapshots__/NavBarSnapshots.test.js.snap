// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  className="navBar"
>
  <div
    className="Logo"
  >
    Math Magicians
  </div>
  <ul
    className="links"
  >
    <a
      className="link"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="link"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="link"
      href="/quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </ul>
</div>
`;
